### DISCUSSION TITLE
Persistence of vision in async mode ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#1055] [Q&A] [answered]
I tried the Seven-Segment display with a circuit running in the async mode. It seems to me that the persistence of vision option is not working. Setting the persistenc of vision value to zero, and enabling the display, the segments light up as required. But for any other value > 0 there is no persistence effect, both in the anode and cathode option as common connection. As clock I used the one from the example of the muller-pipeline. Is this intended ? Or what should I do to make it work ? Note that working with a clock from the Component / IO menu all works well.

**Comment by hneemann** [comment, marked as answer]
Hmm! You are right! The persistence of vision algorithm does not work if there is no clock in the circuit, because the persistence time is measured in clock cycles.
But if you build your own clock, why not use the built-in clock?

**Reply by jongena** [reply]
Many thanks for this remark, this is perfect ! I had the wrong impression that I could not use the 'normal' clock in asynchronous mode !
